src/main/java/com/flightstats/hub/app/SingleHubBindings.java

```diff
@@ -44,4 +32,11 @@ protected void configure() {
     public static Dao<Webhook> buildWebhookDao(WatchManager watchManager, FileWebhookDao dao) {
         return new CachedDao<>(dao, watchManager, "/webhooks/cache");
     }
+
+    @Override
+    protected void configure() {
```

Comment:
Same comment here about the configure. I just realized it's probably because the others are `static`, but I guess I'm not a stickler for ordering by that so much as ordering by what I see as importance/ergonomics? Dunno. I guess it doesn't matter that much.